client/src/main/resources/templates/js/main.js

```diff
@@ -330,17 +330,17 @@ function displaySource(src, title) {
   }
 
   function generateInnerTree(id, key, type, uniqueId) {
-    var leaf = '';
-    if (type === 'className') {
-        leaf = 'data-jstree=\'{"icon":"glyphicon glyphicon-leaf"}\'';
+    var leaf = "";
+    if (type === "className") {
+        leaf = "data-jstree=\"{'icon':'glyphicon glyphicon-leaf'}\"";
     }
     //Remove dot and spaces from the key
-    var id1 = id.replace(/\./g, '').replace(/\s+/g, '');
-    var newId = key.replace(/\./g, '').replace(/\s+/g, '') + type + uniqueId;
-    if ($(id1 + ' ul').length === 0) {
-        $(id1).first().append('<ul><li id="' + newId + '" data-value="' + key + '" data-type="' + type + '" ' + leaf + '>' + key + '</li></ul>');
+    var id1 = id.replace(/\./g, "").replace(/\s+/g, "");
+    var newId = key.replace(/\./g, "").replace(/\s+/g, "") + type + uniqueId;
+    if ($(id1 + " ul").length === 0) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l341)